T.A.Random Colors/index.css

```diff
@@ -0,0 +1,27 @@
+button {
+	padding: 15px 32px;
+	text-align: center;
```

Comment:
Add the project to netlist and share link.

Reply:
https://ecstatic-meitner-327dd5.netlify.app